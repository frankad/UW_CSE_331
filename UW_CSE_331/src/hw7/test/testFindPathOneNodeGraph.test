# Test a case to find path from a graph with one node, oneNodeGraph
# in other words, this test can cover for the case the path from  a
# given node to itself.

# Load oneNodeGraph.tsv
LoadGraph oneNodeGraph oneNodeGraph.tsv

# Find the minimum-cost path from node Eva to itself
FindPath oneNodeGraph Eva Eva 