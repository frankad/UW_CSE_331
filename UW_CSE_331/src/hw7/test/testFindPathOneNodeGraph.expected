# Test a case to find path from a graph with one node, oneNodeGraph
# in other words, this test can cover for the case the path from  a
# given node to itself.

# Load oneNodeGraph.tsv
loaded graph oneNodeGraph

# Find the minimum-cost path from node Eva to itself
 path from Eva to Eva:
total cost: 0.000